docs/blog/2019-07-26-how-i-used-theme-ui-to-build-my-gatsby-themes-library/index.md

```diff
@@ -0,0 +1,176 @@
+<figcaption>Inspirational screenshots</figcaption>
+</figure>
+
+I really liked the prominent and actionable hero section of the Refactoring UI website, the shapes and colors from the other designs. Speaking of Refactoring UI: For the design (and later for coding) I used the TailwindCSS colors and styles.
+
+The design you see live on the website now is pretty much the design I created in Figma — which speaks for the ease of use of a Figma (+ Tailwind styles) => Theme UI (+ Tailwind preset) workflow!
+
+As mentioned I also wanted a rad hero section, nothing boring but also not something that is overused (and potentially can also slow down your website) like particles.js. Since I'm pretty proficient with Cinema4D and 3D in general I just tried to visualize "Gatsby Themes" with 3D models. I drew some inspiration from [Kyle Gill's illustrations](https://kylegill.com/blog/2019-04-22-a-simple-guide-to-gatsby-themes/index). I rendered the scene with Arnold, modified it a bit with Photoshop and et voilá: You can see the finished result on my social preview image now.
+
+<figure>
+<img src="./images/gatsby-themes-3d-illustration.png" alt="3D Rendering showing a little factory receiving input from four objects." />
+<figcaption>Cinema4D Rendering of a "Gatsby Themes factory"</figcaption>
+</figure>
+
+## Code
+
+The website lives inside the [www](https://github.com/LekoArts/gatsby-themes/tree/master/www) folder of the monorepo and thus is already setup to be used with TypeScript. You can have a look at the source code to see how it's done (as going through the complete code would be a bit boring...). A follow-up blogpost will explain how I created the 3D element in the hero section.
+
+I want to focus on [Theme UI](https://theme-ui.com/) and explain how I used it on this landing page, especially the breakpoints syntax, variants, and Tailwind preset. These are truly game changers for me!
+
+To get started I installed the preset and put it into my shadowed theme config (see [official documentation](https://theme-ui.com/presets) on that). Then I had to override the colors e.g. the primary and secondary color. For that I took a look into my Figma design:
+
+<figure>
+<img src="./images/figma-theme-ui-handoff.png" alt="Screenshot showing both Figma and VS Code. The value used in Figma can easily get transferred to VS Code and Theme UI." />
+<figcaption>Using Tailwind styles in both Figma and Theme UI is the best!</figcaption>
+</figure>
+
+It's great to have the same design tokens in all places. I chose the color **Indigo.600** and later in Theme UI I just had to use `colors.indigo[6]`.
+
+### Variants
+
+The bottom view of VS Code shows a snippet of the button variants — variants are a Theme UI feature (see [documentation](https://theme-ui.com/guides/variants)). I used them for the buttons and gradients. The latter looks something like this:
```

Comment:
> The bottom view of VS Code

Where is this? Am I missing it?

Reply:
I'm referring to the figma-theme-ui-handoff screenshot above. Seems like it needs some clarification

Reply:
Perfect, thanks!